api-tests/src/utils/jwt-decoder.ts

```diff
@@ -0,0 +1,12 @@
+import * as jose from "jose";
+import { VcJwtPayload } from "../types/external-api.js";
+
+export const decodeCredentialJwts = (jwts: string[]) => {
+  const mappedJwts: Record<string, VcJwtPayload> = {};
+  jwts.forEach((jwt) => {
+    const claims = jose.decodeJwt(jwt) as VcJwtPayload;
```

Comment:
IMO it would be safer and more natural to specify the generic parameter here: `const vc = jose.decodeJwt<VcJwtPayload>(jwt)`